cmd/snap/cmd_snap_op.go

```diff
@@ -69,8 +69,9 @@ func wait(client *client.Client, id string) (*client.Change, error) {
 			case t.ID == lastID:
 				pb.Set(float64(t.Progress.Done))
 			default:
-				var pkgname string
-				chg.Get("snap-name", &pkgname)
+				var pkgnames []string
+				chg.Get("snap-names", &pkgnames)
+				pkgname := pkgnames[0]
```

Comment:
Can you please add logic here which is conservative, similar to what we see below, so that a bug on our end becomes a polite error rather than a panic?